src/addScores.js

```diff
@@ -0,0 +1,17 @@
+class Score {
+  constructor(name, value) {
+    this.name = name;
+    this.value = value;
+  }
+}
+export default function addScores() {
```

Comment:
- [ ] Ensure the usage of arrow functions instead of the function keyword.

Reply:
@EricMbouwe, Bro I won't report this but do you realize that you've stopped my project not only for something that was not required but also for something that was explicitly said to not be used (API). However thank you for the valuable hints you've given to me. Cheers
![image](https://user-images.githubusercontent.com/83329154/136067500-adcba858-dee3-4c54-bb79-bdec4afd6274.png)

Reply:
Hey @AmineHLub, something was not correct with the project type and I have reviewed your project as the next one instead of the first milestone. Moreover your PR title emphasis that issue, it referred to adding functionalities which is not the point in this project. Anyway thank you for pointing that out, I will do a re-review of your project.

Reply:
@EricMbouwe Thank you very much and I apologize. If anything I thought "You should develop a first working version of the app following a wireframe, but without styling - just focus on functionality." meant that I should add the adding function except for doing it without using any API

Reply:
No worries, all of us make mistakes, that's my fault, I should have paid more attention to your PR before reviewing it.